client/components/account-balances/balance-tooltip.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import { sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Tooltip from 'components/tooltip';
+import {
+	documentationUrls,
+	fundLabelStrings,
+	fundTooltipStrings,
+	learnMoreString,
+} from './strings';
+import { balanceType } from './balance-block';
+
+type BalanceTooltipProps = {
+	type: balanceType;
+	delayDays?: number;
+	isNegativeBalance?: boolean;
+};
+
+const BalanceTooltip: React.FC< BalanceTooltipProps > = ( {
+	type,
+	delayDays,
+	isNegativeBalance,
+} ) => {
+	let content = null;
+	const tooltipButtonLabel = `${ fundLabelStrings[ type ] } tooltip`;
+
+	if ( type === 'available' ) {
+		content = (
+			<>
+				{ fundTooltipStrings.available }{ ' ' }
+				<a
+					target="_blank"
+					rel="noopener noreferrer"
+					href={ documentationUrls.depositSchedule }
+				>
+					{ learnMoreString }
+				</a>
+			</>
+		);
+	}
+
+	if ( type === 'available' && isNegativeBalance ) {
+		content = (
+			<a
+				target="_blank"
+				rel="noopener noreferrer"
+				href={ documentationUrls.negativeBalance }
+			>
+				{ fundTooltipStrings.availableNegativeBalance }
+			</a>
+		);
+	}
+
+	if ( type === 'pending' ) {
+		content = (
+			<>
+				{ sprintf( fundTooltipStrings.pending, delayDays ) }{ ' ' }
+				<a
+					target="_blank"
+					rel="noopener noreferrer"
+					href={ documentationUrls.depositSchedule }
+				>
+					{ learnMoreString }
+				</a>
+			</>
+		);
+	}
+
+	if ( type === 'reserved' ) {
+		content = (
+			<>
+				{ fundTooltipStrings.reserved }{ ' ' }
+				<a
+					target="_blank"
+					rel="noopener noreferrer"
+					href={ documentationUrls.reservedFunds }
+				>
+					{ learnMoreString }
+				</a>
+			</>
+		);
+	}
```

Comment:
Agree with @shendy-a8c comment – can we make the tooltip component dumb and just pass in the content that it should display?

Reply:
Thanks, @shendy-a8c, @haszari this makes sense. I'll rework this and see if I can simplify it.

Reply:
I've changed this up in 1649288fbee7bb9754f2da54d72efccdfabc5047.

I moved the logic for determining the tooltip content up into the `AccountBalancesTabPanel` component, which reduces the amount of prop drilling required.

Let me know what you think 🙏 

I could move the `type` prop out too, but that would start to introduce duplicate logic.

Reply:
Is `type` for behaviour or content? It looks like it's used as a [constant / array index](https://github.com/Automattic/woocommerce-payments/blob/8d62900a1137f065a562a38f54c72d361c1b7f3a/client/components/account-balances/balance-block.tsx#L63) AND [rendered in the UI as a tooltip](https://github.com/Automattic/woocommerce-payments/blob/8d62900a1137f065a562a38f54c72d361c1b7f3a/client/components/account-balances/balance-block.tsx#L56).

I'd suggest passing the content in direct – e.g. pass the title and the tooltip as strings. As it stands the `BalanceBlock` component is coupled to `fundLabelStrings`; IMO that coupling would be better higher up.

Reply:
Thanks @haszari. I've updated BalanceBlock and BalanceBlockTooltip to accept content from the TabPanel component in cf4908d0f7a83d0f7c9d9b61f47a32ce8e2f224c.

`type` is now being used only for generating a unique ID  for each type & currency to be used by aria-labelledby.cf4908d0f7a83d0f7c9d9b61f47a32ce8e2f224c.

Reply:
Sounds like type is now `labelledby` or `id` or `accessibilityparent` or something – maybe that param can be 'dumber' :)

Reply:
Updated in 3ec9aacfcd99814026a0d3ea6d8fdffa7c7327e1.

Reply:
Gotcha, I can see why a generic `balancelabel` or similar might be a better fit.